### PR TITLE
Import UR robots and NPC from gems

### PR DESCRIPTION
**Pull request by jhanca-robotecai**
- import UR Robots and NPC (HumanWorker) from external gems
- remove obsolete assets
- fix few typos in readme

The human trajectory changed compared to the previous configuration. In particular, _DemoLevel1_ adds three more waypoints to NPC1 as _overrides_, while _DemoLevel2_ and _DemoLevel3_ don't. In addition, the latter translates _EndPoint13_ to another position. Keeping this PR as a draft for further checks. 
@michalpelka @pawelbudziszewski please confirm if the differences are caused by a mistake or on purpose.

### CODE REVIEW

**Comment by jhanca-robotecai** [comment]
I double-checked the trajectories of NPCs and it looks correct.

File: README.md [hunk 1]
```diff
@@ -166,7 +182,7 @@ source install/setup.bash
 ```
 The source command needs to be done in the same console where you build and run O3DE.
 
-Assuming that [project's repo](https://github.com/RobotecAI/ROSCon2023Demo) was cloned to `${WORKDIR}`:
+Assuming that a [project's repo](https://github.com/RobotecAI/ROSCon2023Demo) was cloned to `${WORKDIR}`:
```
Comment (arturkamieniecki) [inline README.md:185]:
```suggestion
Assuming that the [project's repo](https://github.com/RobotecAI/ROSCon2023Demo) was cloned to `${WORKDIR}`:
```

**Review by arturkamieniecki — APPROVED** [review]
Looks good and works. There are some warnings in the DemoLevel2.

**Comment by jhanca-robotecai** [comment]
> There are some warnings when launching the DemoLevel2: image

The warnings date back to `main` branch and are not introduced by this PR. I removed the _overrides_ causing the issues in [eef328b](https://github.com/RobotecAI/ROSCon2023Demo/pull/264/commits/eef328b915b5c08cb39f4d041b9f29ad3b9b8e92), hard to spot what the overrides were doing without a bisect.

**Comment by jhanca-robotecai** [comment]
Force-push after merging #265 and fixing conflicts

**Pull request merged**